config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Policy.java

```diff
@@ -265,6 +265,7 @@ public String getType(){
         return EntityTypes.POLICY_TYPE;
     }
 
+    @JsonIgnore
     @Override
     public String getShortenedType() {
```

Comment:
This method was added specifically for this naming. If not required anymore, we can remove it from the interface itself.

Reply:
I think Shortened-Type is for internal use. No need to reflect into our configuration.

Reply:
Oops, we are not using it anywhere. Let me remove this method.

Reply:
Done.